### ISSUE TITLE
Kind of critical that it's missing default

### ISSUE THREAD

**Issue by esobczak1970** [tabulate/tabulate#36]
Any chance you could add the ability to set a default on a field that's kind of a critical piece that's missing. Great plugin otherwise but would be hard to use without defaults. Though I can probably go over into mysqladmin and set those for now.

Thanks,

Eric

**Comment by samwilson** [comment]
You're correct, you can set default values in whatever MySQL management tool you use. But yeah, it'd be great if we could specify these within Tabulate. I probably won't have much time to work on this for the next few weeks though, sorry.

**Comment by esobczak1970** [comment]
Ok will try to work around it for now but that would really help. Though I know how it is if only the world would give us the time we need. :-)

Eric

-----Original Message-----
